internal/sessiontest/helper_servers_test.go

```diff
@@ -229,19 +222,7 @@ func IrmaServerConfiguration() *server.Configuration {
 			revKeyshareTestCred: {RevocationServerURL: revocationServerURL},
 		},
 		JwtPrivateKeyFile: jwtPrivkeyPath,
-		StaticSessions: map[string]interface{}{
-			"staticsession": irma.ServiceProviderRequest{
-				RequestorBaseRequest: irma.RequestorBaseRequest{
-					CallbackURL: staticSessionServerURL,
-				},
-				Request: &irma.DisclosureRequest{
-					BaseRequest: irma.BaseRequest{LDContext: irma.LDContextDisclosureRequest},
-					Disclose: irma.AttributeConDisCon{
-						{{irma.NewAttributeRequest("irma-demo.RU.studentCard.level")}},
-					},
-				},
-			},
-		},
+		StaticSessions:    map[string]interface{}{},
```

Comment:
Why is this line here? Can we not just remove it, and instantiate a new `map[string]interface{}` in `testStaticQRSession()`?

Reply:
That's possible too indeed. I'll change it.